source/MaterialXContrib/MaterialXMaya/CreateMaterialXNodeCmd.cpp

```diff
@@ -161,7 +161,7 @@ MStatus CreateMaterialXNodeCmd::doIt( const MArgList &args )
         }
 
         mx::DocumentPtr document = MaterialXUtil::loadDocument(
-            documentFilePath.asChar(), Plugin::instance().getLibrarySearchPath()
+            documentFilePath.asChar(), Plugin::instance().getLibraryDocument()
```

Comment:
We now cache the document with libraries in the plugin instance and can import these to per-material documents instead of reloading them every time.